hibike/README.md

```diff
@@ -196,6 +196,21 @@ Device Type Enumeration:
 |         |                | 7            | enable3    | bool       | yes   | yes    |
 |  0x08   | LinearActuator |              |            |            |       |        |
 |  0x09   | ColorSensor    |              |            |            |       |        |
+|  0x0A   | YogiBear       | 0            | enable              | bool       | yes   | yes    |
```

Comment:
Yeah I made that decision.

We found conflicting sources on whether 0x06 was for grizzly or yogibear.  So I went with 0x06 for grizzly, and 0x0A for Yogibear, because
-The README.md file said that 0x06 was taken
-Futureproofing just in case we decide to use the 50+ grizzlies we have laying around.